client/src/app/shared/shared-main/misc/top-menu-dropdown.component.ts

```diff
@@ -94,6 +94,16 @@ export class TopMenuDropdownComponent implements OnInit, OnDestroy {
     this.isModalOpened = false
   }
 
+  onActiveLinkScrollToTop (link: HTMLAnchorElement) {
+    if (!this.isBroadcastMessageDisplayed && this.router.url.replace(/#.+$/, '').includes(link.getAttribute('href'))) {
+      window.scrollTo({
+        left: 0,
+        top: 0,
+        behavior: 'smooth'
```

Comment:
@Chocobozzz here the behavior smooth on scroll here seems to me justified because the goal is to back / scroll to top when clicking on the active link. So though the intended action is a scroll and not navigation I think the smooth can be used. What do you think ?

Reply:
yes but why don't you use the same behaviour here: https://github.com/Chocobozzz/PeerTube/pull/3066/files#diff-e1e01cbde1b933afb4bd4324841ddcfcR206?

Reply:
here it's more for navigation when you click on a link in the user dropdown, but we can also use the smooth behavior here too, it was obvious to follow what the router does in that case but yes the view does not reload so we can uses smooth too.